index.html

```diff
@@ -7,8 +7,10 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link rel="stylesheet" href="styles.css">
         <link rel="shortcut icon" type="image/png" href="/approachable-open-source-logo.png"/>
+        <link rel="alternate" href="https://github.com" hreflang="en-us" />
     </head>
     <body>
+        <a href="#main-content" class="skip-link">Skip to Main Content</a> 
```

Comment:
The skip link tag looks great, but it doesn't actually target anything on the page.

Reply:
To target, do we need to add an `id="main-content"` to the main content below? @MobGG, since it is your Pull Request, do you want to tackle that?